### DISCUSSION TITLE
ICM - Connection log shows Offline

### DISCUSSION THREAD

**Discussion by jcolinger3** [InternetConnectionMonitor/icm-public#117] [Bug reports]
The connection log shows offline even though the tests and realtime show online.
Discrepancy.  Yet, if a Clear Log, the state goes back to Online.
Sort of defeats to the purpose of a Log if it needs to be cleared before it reflects accurate information

![image](https://github.com/user-attachments/assets/4256be18-d859-4f09-937c-3425952249e4)
![image](https://github.com/user-attachments/assets/c8fa2bbb-c34c-4a36-b4c2-f1c5fe82510c)

After Clear Log
![image](https://github.com/user-attachments/assets/00f09741-1ab1-4fab-b61a-726576e7f29e)

**Comment by dmishh** [comment]
@jcolinger3 There may be some other extensions blocking the monitoring. Please, see - #113
